l10n_it_ddt/i18n/it.po

```diff
@@ -702,7 +682,7 @@ msgstr "I Picking selezionati %s hanno Causali di trasporto differenti"
 #: code:addons/l10n_it_ddt/models/stock_picking_package_preparation.py:166
 #, python-format
 msgid "Selected Picking is already linked to DDT: %s"
-msgstr "I Picking selezionati sono già collegati al DDT: %s"
+msgstr "Gli ordini di consegna selezionati sono già collegati al DDT: %s"
```

Comment:
se lo cambi qui, per coerenza, lo cambierei anche negli altri posti in cui figura. `Selected Picking`*.

Reply:
@tafaRU grazie per la review, in realtà quelle traduzioni non sono come sono "entrate". D'ora in poi userò Transifex :-)... come possiamo procedere?

Reply:
trattandosi di poche righe di codice, fai prima a creare una nuova PR in cui al massimo aggiungi la traduzione della nuova stringa.

Reply:
@tafaRU ecco la nuova pr https://github.com/OCA/l10n-italy/pull/471